Dockerfile.cluster-nfd-operator-build

```diff
@@ -1,10 +1,10 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-4.14
+FROM golang:1.20-alpine3.18
 
 ENV GO111MODULE=on
 ENV GOFLAGS=""
 
-RUN yum install -y which
-RUN yum install -y podman
+RUN ["apk", "add", "bash", "make", "docker", "curl", "shadow", "git"]
```

Comment:
Currently there is a problem with using podman and skipper: the podman image are not created on the host internal image repo. Since skipper is used for development and not for official builds, we can use it until the issue is resolved in the skipper